pkg/appgw/ingress_rules.go

```diff
@@ -38,7 +38,8 @@ func (builder *appGwConfigBuilder) processIngressRules(ingress *v1beta1.Ingress)
 		}
 
 		// Enable HTTP only if HTTPS is not configured OR if ingress annotated with 'ssl-redirect'
-		if annotations.IsSslRedirect(ingress) || !httpsAvailable {
+		sslRedirect, _ := annotations.IsSslRedirect(ingress)
```

Comment:
Lets glog an Error here if one is returned

Reply:
Will add in a following PR!